sklearn/gaussian_process/_gpr.py

```diff
@@ -180,6 +180,9 @@ def optimizer(obj_func, initial_theta, bounds):
     0.3680...
     >>> gpr.predict(X[:2,:], return_std=True)
     (array([653.0..., 592.1...]), array([316.6..., 316.6...]))
+
+    See :ref:`sphx_glr_auto_examples_gaussian_process_plot_compare_gpr_krr.py`
+    for an example to compare kernel ridge and Gaussian process regression
```

Comment:
I would move this reference to the general summary at the top of the file just before mentioning the user-guide.

I think that this example is actually didactic to understand the difference between a frequentist approach (`KernelRidge`) and the more Bayesian approach (`GaussianProcessRegressor`).

I think that we should rephrase as:

```python
    To learn the difference between a point-estimate approach vs. a more
    Bayesian modelling approach, refer to the example entitled
    :ref:`sphx_glr_auto_examples_gaussian_process_plot_compare_gpr_krr.py`.
```

Reply:
I made the suggested changes in `GaussianProcessRegressor`, can you please review? Thanks!